app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,6 @@
  *= require foundation_and_overrides
 
  */
+ li {
+   list-style: none;
+ }
```

Comment:
Best practice for any custom styling is they should go in a new file within the `stylesheets` folder, this manifest file is an absolute last resort.
While we're here, we also want to modify the order of require statements lines 13-15 (they are generated this way by make_it_so when it adds Foundation)
```
 *= require foundation_and_overrides
 *= require_tree .
 *= require_self
```
This makes sure that any base Foundation styles are applied first, `require_tree` requires all files within the stylesheets folder including any files we create, and `require_self` requires this file last